### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by thevan96**
- Init project with rails
- Setup CI
- Issue: https://edu-redmine.sun-asterisk.vn/issues/37704

### CODE REVIEW

**Comment by thevan96** [comment]
ready

image

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>DnOe45Ecomerces</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (vanvtt-0952) [inline app/views/layouts/application.html.erb]: Nháy đôi cu

File: .sun-ci.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    environment:
+      MYSQL_DATABASE: db_test
+      MYSQL_USER: user_test
+      MYSQL_PASSWORD: password_test
+      MYSQL_ROOT_PASSWORD: password_test
+  before_script:
+  - bundle _2.1.2_ install --path vendor/bundle
+  script:
+  - RAILS_ENV=test bundle exec rails db:drop db:create db:migrate
+  - bundle _2.1.2_ exec rspec
```
Comment (vanvtt-0952) [inline .sun-ci.yml]: comment line 34 lại cu nhé

**Comment by thevan96** [comment]
ready

image

**Pull request merged**